styles/styles.css

```
@@ -165,6 +211,23 @@ main {
     margin-right: auto
 }
 
+
+.advantages-photo-first {
+    flex-grow: 0;
+    flex: none;
+}
+
+.advantages-photo-secound {
+    flex-grow: 0;
+    flex: none;
+}
+
+.advantages-service-list {
+    display: flex;
+    justify-content: center;
+}
```

```suggestion
.advantages-service-list {
    display: flex;
}
```
Дочерние элементы в сумме занимают всё пространство - это раз. 
Если их будет не 3, а 2 или 5 центрирование будет выглядеть очень странно.